app/assets/javascripts/account/AccountService.js.coffee

```diff
@@ -51,6 +51,26 @@ AccountService = ($state, $auth, $modal, $http, $translate, ShortFormApplication
         return false
       )
 
+  Service.requestPasswordReset = ->
+    params =
+      email: Service.userAuth.email
+    $auth.requestPasswordReset(params).then((resp) ->
+      Service.userAuth.resetPwdEmailSent = true
+    ).catch (resp) ->
+      # handle error response
+    return
+
+  Service.updatePassword = ->
+    params =
+      password: Service.userAuth.password
+      password_confirmation: Service.userAuth.password_confirmation
+
+    $auth.updatePassword(params).then((resp) ->
+      $state.go('dahlia.my-applications')
```

Comment:
I think this should be `dahlia.my-account`? I didn't see any info in the story instructions as to where it should redirect, but that's the usual place to redirect after login -- we only redirect to `my-applications` after you submit an application (so you can see it in there).